front/components/sparkle/navigation.tsx

```diff
@@ -61,29 +62,33 @@ export const topNavigation = ({
   owner: WorkspaceType;
   current: TopNavigationId;
 }) => {
+  const isOnV2 = isOnAssistantV2(owner);
   const isDust = isDevelopmentOrDustWorkspace(owner);
-  const nav: SparkleAppLayoutNavigation[] = [
-    {
-      id: "assistant",
+  const nav: SparkleAppLayoutNavigation[] = [];
+
+  if (isOnV2) {
+    nav.push({
+      id: "assistants",
```

Comment:
typo?

Reply:
I needed different ids for V1 and V2 when we were displaying both but indeed now we don't need it anymore, thanks for catching!